components/03-modules/sidebar-block/_sidebar-block.scss

```diff
@@ -0,0 +1,111 @@
+$sidebar-block__padding                    : $spacer--medium 0 $spacer--large !default;
+$sidebar-block__heading-margin-bottom      : $spacer !default;
+$sidebar-block__title-font-weight          : $font-weight-bold !default;
+$sidebar-block__list-margin-bottom         : $spacer !default;
+$sidebar-block__item-margin-bottom         : $spacer !default;
+$sidebar-block__index-margin               : 0 $spacer !default;
+$sidebar-block__index-font-weight          : $font-weight-bold !default;
+$sidebar-block__index-min-width            : $spacer--medium !default;
+$sidebar-block__remove-padding             : $spacer--medium !default;
+$sidebar-block__remove-margin-right        : $spacer !default;
+$sidebar-block__remove-icon-margin         : 0 auto !default;
+$sidebar-block__remove-icon-padding        : 0 !default;
+$sidebar-block__icon-fill                  : $red !default;
+$sidebar-block__icon-size                  : $spacer--medium !default;
```

Comment:
icon size should not get value from spacer. Please use a value in px's.

Reply:
Done